### DISCUSSION TITLE
Name of the software: "CAFE5", "CAFE 5", or just "CAFE"?

### DISCUSSION THREAD

**Discussion by heavywatal** [hahnlab/CAFE5#188] [Q&A] [answered]
How should users mention this software on papers and other contexts? The occurrences in `README.md` are inconsistent.

For example, "2" in "ggplot2" and "DESeq2" is not a version, but a part of the names. "MAFFT" and "BLAST" are usually mentioned without version number (except in the methods section). How about CAFE?

**Comment by benfulton** [comment, marked as answer]
We haven't really thought about this. Let's use "CAFE 5" going forward, but I can't guarantee we will be all that consistent about it.
